server/checks/Specification.js

```diff
@@ -0,0 +1,110 @@
+import Check from './Check'
+import { logger } from '../../common/debug'
+import * as EVENTS from '../model/GithubEvents'
+
+const CHECK_TYPE = 'specification'
+const ACTIONS = ['opened', 'edited', 'reopened', 'synchronize']
+
+const DEFAULT_REQUIRED_LENGTH = 8
+const ISSUE_PATTERN = /^[^#\s]*#\d+$/
+// Grubber's pattern
+const URL_PATTERN = /\b((?:[a-z][\w-]+:(?:\/{1,3}|[a-z0-9%])|www\d{0,3}[.]|[a-z0-9.\-]+[.][a-z]{2,4}\/)(?:[^\s()<>]+|\(([^\s()<>]+|(\([^\s()<>]+\)))*\))+(?:\(([^\s()<>]+|(\([^\s()<>]+\)))*\)|[^\s`!()\[\]{};:'".,<>?«»“”‘’]))/i
+
+const info = logger(CHECK_TYPE, 'info')
+const error = logger(CHECK_TYPE, 'error')
+
+const status = (description, state = 'success') => ({
+  description,
+  state,
+  context: 'zappr/specification'
```

Comment:
Coming to think about it, in the commit message check we use the context `zappr/commit/message` to account for possible future checks that also use the commit, ie if it was signed could then use `zappr/commit/signed`. What do you think of using `zappr/pr/specification` or `zappr/pr/spec` as context to signal the resource it works on to the user? Does it even make a difference?

These kind of questions sound mundane, but we can't change it later without breaking people's configurations :)